web/src/filter.ts

```diff
@@ -536,6 +551,8 @@ export class Filter {
                 return verb + "channels";
             case "near":
                 return verb + "messages around";
+            case "with":
+                return verb + "Thread with Message";
```

Comment:
Capitalization here is not in line with the standard for this file. I think you want something like "conversation containing" here.

Reply:
updated!